internal/cmd/tracking-issue/main.go

```diff
@@ -230,19 +303,42 @@ func Assignee(assignees []string) string {
 
 type TrackingIssue struct {
 	*Issue
-	Issues []*Issue
-	PRs    []*PullRequest
+	Issues         []*Issue
+	PRs            []*PullRequest
+	LabelWhitelist []string
```

Comment:
Looking at the uses of this field, it would benefit from being a set, rather than a list, to optimize set inclusion.
```suggestion
	LabelWhitelist map[string]struct{}
```

Reply:
i want to preserve the order of labels so that users can specify the order their labels will be rendered in